src/AccountingAPIClient.ts

```diff
@@ -126,7 +126,20 @@ export class AccountingAPIClient extends BaseAPIClient {
 			const endpoint = 'banktransactions' + generateQueryString(args, true);
 			return this.oauth1Client.post<BankTransactionsResponse>(endpoint, bankTransaction);
 		},
-		attachments: this.generateAttachmentsEndpoint('banktransactions')
+		attachments: this.generateAttachmentsEndpoint('banktransactions'),
+		history: {
+			get: async (args?: { BankTransactionID: string }): Promise<HistoryResponse> => {
```

Comment:
Just looking at the docs for H&N. I think the ID Guid is required. 

What do you think about making `args` required in all these endpoints? Aka remove the `?`

Reply:
Eg: 

`get: async (args?: { BankTransactionID: string }): Promise<HistoryResponse> => {`
becomes:
`get: async (args: { BankTransactionID: string }): Promise<HistoryResponse> => {`

Reply:
Yes - that's correct - I've not used TypeScript before so I missed that. Shall I change and resubmit?

Reply:
No worries.

Yup, just make the changes and push the new commit to your `master` the pull request will update automatically.

Reply:
Done